### PR TITLE
fixed error if dune-fem is buildt with polygongrid

### PR DESCRIPTION
**Pull request by hnil**
- this added need finding polygongrid.
- if make the dune-fem in opm consistent with optional dependencies

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: cmake/Modules/Finddune-polygongrid.cmake [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  ""
+
+  # test program
+"#include <dune/polygongrid/mesh.hh>
+int main (void) {
+   return 0;
+}
+"
+# config variables
+# config variables
```
[the remark this answers is not in the document]
Comment (hnil) [reply cmake/Modules/Finddune-polygongrid.cmake]: Fixed.

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — COMMENTED** [review]
I think this looks good, but I don't really have any experience with this grid manager.  That said, the build check is successful so unless anyone objects, I'm going to merge this into the master branch no later than noon CET tomorrow (Friday 17 March).

**Pull request merged**